client/my-sites/people/people-profile/index.jsx

```diff
@@ -77,12 +79,12 @@ class PeopleProfile extends Component {
 		return text;
 	};
 
-	getRoleBadgeClass = ( role ) => {
-		role = 'undefined' === typeof role ? this.getRole() : role;
+	const getRoleBadgeClass = ( role ) => {
+		role = 'undefined' === typeof role ? getRole() : role;
```

Comment:
This can be written more conveniently as `getRoleBadgeClass = ( role = getRole() ) => {`. Same in the `getRoleBadgeText` method.